app/models/concerns/node_properties.rb

```diff
@@ -67,7 +67,7 @@ def has_any_property?
   #
   # Anything else will be saved under the `services_extras` key
   def set_service(data)
-    data.symbolize_keys!
+    data = data.symbolize_keys
```

Comment:
is this for better reading or this was not working?
should we indent the `=` in this line to match the next lines?

Reply:
I see the commit message now `HashWithIndifferentAccess doesn't have the bang operatore symbolize keys.`